.github/workflows/build.yml

```diff
@@ -80,6 +83,11 @@ jobs:
           if [ "${{ runner.os }}" = "Windows" ]; then
             EXTRA_TWISTER_FLAGS="--short-build-path -O/tmp/twister-out"
           fi
+          
+          if [ "${{ runner.os }}" = "Linux" ]; then
+            export PKG_CONFIG_PATH=/usr/lib/i386-linux-gnu/pkgconfig
+          fi
+          
```

Comment:
do you need both this and the one on line 47?

Reply:
Guess not if we only need it once before compiling. Going to try without line 47